src/main/java/frc/robot/Constants.java

```diff
@@ -44,8 +44,21 @@
   }
 
   public static final class DriveTrain {
-    public static final double TRACK_WIDTH = 0.0; // Inches
-    public static final double ENCODER_INCHES_PER_TICK = 1.0;
+    /**
+     * Track width in inches
+     */
+    public static final double TRACK_WIDTH = 0.0;
```

Comment:
Javadoc here is bloated, just use trailing comment

Reply:
resolved in 530b65c